src/main/java/tuteez/commons/util/UiUtil.java

```diff
@@ -0,0 +1,103 @@
+package tuteez.commons.util;
```

Comment:
It's great that you create a UiUtil class to improve code reusability and encapsulation!